### ISSUE TITLE
Basic Tutorial not working

### ISSUE THREAD

**Issue by jbkoh** [immesys/wave#24]
Hi,

I am trying to learn WAVE for my projects. I just downloaded the release version 0.4.0, and ran ``./waved`` to follow the instruction in README.md. However, it showed "Illegal instruction". I tried both with/without sudo. My OS is Ubuntu 18.04. I'd appreciate if you have any clue about this. 

Thanks in advance.

**Comment by gtfierro** [comment]
I think you are better off building from source, or trying different builds. There are a number of things that have changed between 0.4.0 and the current commit. On [my fork](https://github.com/gtfierro/wave) I've tried to automate some builds that you can try (v0.5-alpha, for amd-linux64), but I also have a Makefile that you can use to see the instructions for building. You'll need `godep` installed to handle the dependencies.

**Comment by jbkoh** [comment]
@gtfierro Thanks for the advice. I successfully compiled waved and wv from the source code based on your Makefile. Now the entire Tutorial is working. 

There is a small fix for the usability. ``waved`` and ``wv`` use different ports by default. I had to manually change the port in ``wave.toml`` in the release version.

Also,  in the default ``wave.toml``, "https://standalone.storage.bwave.io/v1" is defined for ``storage``. What is it for and is it okay to leave it as it is?

**Comment by gtfierro** [comment]
I guess those got changed at some point... they certainly used to be the same! 

The storage is covered in detail in the USENIX Security paper. It is where WAVE stores attestations. The default value should work fine

**Comment by jbkoh** [comment]
Does it mean that all the attestations that I request are stored in https://standalone.storage.bwave.io/v1? 

What should I do if I want to run the database by myself?

Also, currently ``storage.default`` seems to be activated and ``storage.vldm`` is commented out. What are the differences between them?

**Comment by gtfierro** [comment]
Yes, they are all stored there. You can dig into the `storage/` folder if you want to run a storage server yourself, but its likely not worth the trouble.

The paper documents the differences between the default and VLDM storage. The default one is activated so that you don't have to worry about this.

**Comment by jbkoh** [comment]
- I can't find VLDM in the paper. Is it same as ULDM?
- What is the "default" server? Sorry I couldn't locate its description in the paper either. I looked up evaluation to see how the authors evaluate VLDM with a baseline (which I thought was the default one,) but it's not there. Maybe I missed it as I just skimmed through it.
- What's the plan to make the storage deployable? As a user, I'd prefer fully controling my deployment.

**Comment by shrBest** [comment]
Hello, I‘m a student who is studying this project recently, but the project deployment process is not smooth. The main problem is the storage server website [https://standalone.storage.bwave.io/v1]has been unable to connect. Just like the communication between you and jbkoh, is it almost impossible to deploy Wave locally?

Looking forward to your reply！
